Setup/Methods/Select.php

```diff
@@ -77,15 +77,38 @@ public function __toString(): string
     {
         return trim(
             sprintf(
-                "SELECT %s%s FROM %s%s%s%s%s%s",
+                'SELECT %s%s FROM %s%s%s%s%s%s',
                 true === $this->distinct ? 'DISTINCT ' : '',
```

Comment:
niet op true checken, je types moeten goed genoeg zijn om dit niet nodig te maken, dus gewoon zo
```suggestion
                $this->distinct ? 'DISTINCT ' : '',
```